image/templates/helm/stackrox-central/templates/02-config-controller-03-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+{{- include "srox.init" . -}}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: config-controller
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: config-controller
+    {{- include "srox.labels" (list . "deployment" "config-controller") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "deployment" "config-controller") | nindent 4 }}
+spec:
+  replicas: 1
+  minReadySeconds: 15
+  selector:
+    matchLabels:
+      app: config-controller
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      namespace: {{ .Release.Namespace }}
+      labels:
+        app: config-controller
+        {{- include "srox.podLabels" (list . "deployment" "config-controller") | nindent 8 }}
+      annotations:
+        {{- include "srox.podAnnotations" (list . "deployment" "config-controller") | nindent 8 }}
+        {{- if eq ._rox.env.openshift 4 }}
+        openshift.io/required-scc: nonroot-v2
```

Comment:
(same as above) I don't think the config-controller needs to use the `nonroot-v2` SCC.